### PR TITLE
Update cn.h

### PR DESCRIPTION
**Pull request by giwhub**

### CODE REVIEW

**Comment by GrayHatter** [comment]
@giwhub the `//` at the start of each line means `this is a comment, ignore everything after` so if you want to enable those translations, you need to remove the `//` at the start of each line.

**Comment by giwhub** [comment]
thank you.

File: langs/cn.h [hunk 1]
```diff
@@ -151,13 +150,11 @@ msgstr("代理 (SOCKS 5)")
 msgid(WARNING)
 msgstr("修改网络或代理设置将会断开tox网络连接")
```
Comment (stal888) [inline langs/cn.h]: "Tox"?

**Comment by stal888** [comment]
@GrayHatter I can't read Chinese but I went over it with rikaikun and it looks ok to me.

**Comment by GrayHatter** [comment]
thanks @giwhub @stal888

**Pull request merged**